clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -866,50 +894,93 @@ def _get_fileset_catalog(self, catalog_ident: NameIdentifier):
         finally:
             write_lock.release()
 
-    def _get_filesystem(self, actual_file_location: str):
+    # Disable Too many branches (13/12) (too-many-branches)
+    # pylint: disable=R0912
+    def _get_filesystem(
+        self,
+        actual_file_location: str,
+        fileset_catalog: Catalog,
+        name_identifier: NameIdentifier,
+    ):
         storage_type = self._recognize_storage_type(actual_file_location)
         read_lock = self._cache_lock.gen_rlock()
         try:
             read_lock.acquire()
-            cache_value: Tuple[StorageType, AbstractFileSystem] = self._cache.get(
-                storage_type
+            cache_value: Tuple[int, AbstractFileSystem] = self._cache.get(
+                name_identifier
             )
             if cache_value is not None:
-                return cache_value
+                # if the cache value is not expired, return the cache value
+                if cache_value[0] > time.time() * 1000:
+                    return cache_value[1]
         finally:
             read_lock.release()
 
         write_lock = self._cache_lock.gen_wlock()
         try:
             write_lock.acquire()
-            cache_value: Tuple[StorageType, AbstractFileSystem] = self._cache.get(
-                storage_type
+            cache_value: Tuple[int, AbstractFileSystem] = self._cache.get(
+                name_identifier
             )
+
             if cache_value is not None:
-                return cache_value
+                # if the cache value is not expired, return the cache value
+                if cache_value[0] > time.time() * 1000:
```

Comment:
process the cases which  expire time is zero? and could you use a function to check whether the token expires?

Reply:
I have handled it. see 
<img width="792" alt="image" src="https://github.com/user-attachments/assets/c25ec5de-6d47-45f3-a6fb-84fcc02d89c9" />


> nd could you use a function to check whether the token expires?

ok

Reply:
for local and HDFS, you return 0 for expire time, suggest defining a constant to represent expire time.

Reply:
OK, I have changed it to a constant value.  I just follow what it did in `S3SecretKeyCredential`, `OSSSecretKeyCredential`, `AzureAccountKeyCredential`, there are all use `0` directly.